src/components/EventToast/useEventToast.tsx

```diff
@@ -25,13 +25,19 @@ function useEventToast() {
       return;
     }
 
-    const provider = getProvider(undefined, chainId);
-    provider.getBlock().then((block) => {
-      const now = Date.now() / 1000;
-      if (now - block.timestamp > 60) {
-        setIsArbitrumDown(true);
-      }
-    });
+    async function helper() {
+      const provider = getProvider(undefined, chainId);
+      const blockNumber = await provider.getBlockNumber();
+
+      provider.getBlock(blockNumber).then((block) => {
```

Comment:
nitpick: it may be prettier to continue using await as it was started, but its just cosmetics